test/specs/init.js

```diff
@@ -29,6 +29,21 @@ module.exports = function (promise) {
       expect(fsm.current).to.be.equal('green');
     });
 
+    it('should accept an Object for events property', function () {
+      StateMachine.Promise = promise;
+
+      var fsm = StateMachine({
+        events: {
```

Comment:
Although it is more readable to have the events as an `object` is not on par feature wise with having it as `array` (e.g. not being able to have multiple events with same name). For this reason we cannot say that these formats are interchangeable. 

However, this is a library and we should maintain a level of flexibility. What about having an `eventsLite` configuration property that is merged into the events `array`? This way the users have a clear idea about what are the limitations of this approach.

Although I like the `object` notation, my main concern here is how to avoid confusion between the two. Do yu have a better way of addressing this issue?